pkgs/by-name/hy/hyprland/package.nix

```diff
@@ -164,9 +134,8 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optionals enableXWayland [
       xorg.libxcb
       xorg.libXdmcp
-      xorg.xcbutil
       xorg.xcbutilerrors
-      xorg.xcbutilrenderutil
+      # xorg.xcbutilrenderutil
```

Comment:
any reason to leave this in here? if so maybe a comment would be helpful.

Reply:
Oh, I forgot to remove that. I was testing which packages were still needed.